### ISSUE TITLE
[BUG] Problem with Install kodi addons

### ISSUE THREAD

**Issue by The-cpu-max** [jurialmunkey/skin.arctic.fuse#151]
### Skin section

Home Widgets

### Current Behavior

cant to install any from the repo on new kodi

### Expected Behavior

install any addon form the repo

### Steps To Reproduce

go to the repo and install

### Screenshots and Additional Info

![image](https://github.com/jurialmunkey/skin.arctic.fuse/assets/102416233/dc487bfd-bcc3-45c5-94da-f771b2ddc06b)
![image](https://github.com/jurialmunkey/skin.arctic.fuse/assets/102416233/388f1860-0cad-41b5-b5db-2d48ddfbb29d)

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
You must enable setting to allow installing official addons from any repo.

**Comment by jurialmunkey** [comment]
Follow ALL the instructions on the forum

1. Enable "Unknown Sources" in Kodi Settings > System > Add-ons
2. Enable "Update official add-ons from: Any repositories" in Kodi Settings > System > Add-ons
3. Install my repository using either the zip or file manager source
4. Install the skin from my repo: Jurialmunkey Alpha Repo > Look and Feel > Skins > Arctic Fuse

**Comment by The-cpu-max** [comment]
> Follow ALL the instructions on the forum
> 
> 1. Enable "Unknown Sources" in Kodi Settings > System > Add-ons
> 2. Enable "Update official add-ons from: Any repositories" in Kodi Settings > System > Add-ons
> 3. Install my repository using either the zip or file manager source
> 4. Install the skin from my repo: Jurialmunkey Alpha Repo > Look and Feel > Skins > Arctic Fuse

that wha

https://github.com/jurialmunkey/skin.arctic.fuse/assets/102416233/bb07b754-bfd6-4974-9a08-d53b7516c3a3

t i do but its not work

**Comment by jurialmunkey** [comment]
Then install the dependencies manually

**Comment by The-cpu-max** [comment]
> Then install the dependencies manually

It works manually
But check exactly what the problem is there
It shouldn't be like this

**Comment by jurialmunkey** [comment]
There is nothing to check. My repo works fine.

If you manually install dependencies via zip then Kodi will not update them. You need to go into addon settings and use the version button to switch the dependency back to tracking the version in my repo.

I cannot remotely change your own Kodi settings. If change things to install manually you can't expect them to install automatically.

**Comment by The-cpu-max** [comment]
> There is nothing to check. My repo works fine.
> 
> This is a problem with your settings.
> 
> If you manually install dependencies via zip then Kodi will not update them automatically. You need to go into the settings for each addon you installed manually and use the version button to switch the dependency back to tracking the version in my repo.
> 
> If you set an addon to install manually I can't change your settings. That's your decision not mine.

But when I try to install directly from the repo on clean Kodi - it writes this error
You can try it yourself and see if it gives the same error
Only after you install the dependencies manually it works

**Comment by jurialmunkey** [comment]
>  You can try it yourself and see if it gives the same error

I have tried myself. It works correctly. I use the repo on all my machines in my household and have updates without any issue.

Read what I wrote. If you manually install an addon then it no longer TRACKS the repo. You then need to go to the addon info page and press the VERSIONS button and select the version from the repo.

I CANNOT fix your settings.

**Comment by jurialmunkey** [comment]
> on clean Kodi

If you are truly on a clean Kodi then you have followed all the steps.

More likely however is that you simply reinstalled Kodi over the top. This IS NOT a clean version of Kodi because the Kodi userdata folder is kept intact.

**Comment by The-cpu-max** [comment]
> > on clean Kodi
> 
> If you are truly on a clean Kodi then you have NOT followed all the steps I listed above about changing settings to allow installing from any repository.
> 
> More likely however is that you simply reinstalled Kodi over the top of an old install. This IS NOT a clean version of Kodi because the Kodi userdata folder is kept intact. You must delete the entire userdata folder to have a clean install.

But I deleted the entire appdata folder including everything
it does not help
kodi is absolutely clean
I followed the instructions

**Comment by jurialmunkey** [comment]
My loungeroom pc is setup to install repo updates automatically, so I know my repo works correctly as otherwise I would not have the most recent version of the skin.

Typically this issue with failing to install dependencies is because you have not followed the instruction to allow installing updates from any repository.

**Comment by jurialmunkey** [comment]
I just installed from a portable fresh install with no difficulty.

Show me a screenshot of your settings on this page:
Settings > System > Addons

**Comment by The-cpu-max** [comment]
> I just installed from a portable fresh install with no difficulty.
> 
> Show me a screenshot of your settings on this page: Settings > System > Addons
![image](https://github.com/jurialmunkey/skin.arctic.fuse/assets/102416233/50996c20-034a-42d9-ac6f-dd7a99387f70)
![image](https://github.com/jurialmunkey/skin.arctic.fuse/assets/102416233/ffb70e20-e445-404e-a22b-3b2de760f878)

**Comment by The-cpu-max** [comment]
https://github.com/jurialmunkey/skin.arctic.fuse/assets/102416233/eaac7c10-df0b-4c39-b582-a7f747b0353d

**Comment by jurialmunkey** [comment]
![image](https://github.com/jurialmunkey/skin.arctic.fuse/assets/4618719/f876bd1b-b095-48e9-8df2-a6037133418a)


You did NOT follow all the steps!!!!

### Enable "Update official add-ons from: Any repositories" in Kodi Settings > System > Add-ons

**Comment by The-cpu-max** [comment]
> ![image](https://user-images.githubusercontent.com/4618719/273375758-f876bd1b-b095-48e9-8df2-a6037133418a.png)
> 
> You did NOT follow all the steps!!!!
> 
> ### Enable "Update official add-ons from: Any repositories" in Kodi Settings > System > Add-ons

ok thank you now its work
